Tools/TestWebKitAPI/Tests/WebKitCocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -1465,12 +1466,12 @@ HTTPServer server([parentFrame = String(parentFrame), subFrame = String(subFrame
 
 TEST(WebAuthenticationPanel, PublicKeyCredentialCreationOptionsMinimum)
 {
-    uint8_t identifier[] = { 0x01, 0x02, 0x03, 0x04 };
-    NSData *nsIdentifier = [NSData dataWithBytes:identifier length:sizeof(identifier)];
+    std::array<uint8_t, 4> identifier { 0x01, 0x02, 0x03, 0x04 };
```

Comment:
We can’t simply omit the 4 though, we’d need to omit uint8_t too and then it would deduce the wrong type. There is std::to_array iirc the name but the Sony port doesn’t support it yet.

Reply:
This is Cocoa code so I'll use `std::to_array`